generator/internal/client_generator.cc

```diff
@@ -89,32 +90,35 @@ Status ClientGenerator::GenerateHeader() {
   if (!result.ok()) return result;
 
   // Client Class
-  HeaderPrint(  // clang-format off
-    "\n"
-    "$class_comment_block$\n"
-    "class $client_class_name$ {\n"
-    " public:\n"
-    "  explicit $client_class_name$(std::shared_ptr<$connection_class_name$> connection, Options opts = {});\n"
-    "  ~$client_class_name$();\n"
-    "\n"
-    "  //@{\n"
-    "  // @name Copy and move support\n"
-    "  $client_class_name$($client_class_name$ const&) = default;\n"
-    "  $client_class_name$& operator=($client_class_name$ const&) = default;\n"
-    "  $client_class_name$($client_class_name$&&) = default;\n"
-    "  $client_class_name$& operator=($client_class_name$&&) = default;\n"
-    "  //@}\n"
-    "\n"
-    "  //@{\n"
-    "  // @name Equality\n"
-    "  friend bool operator==($client_class_name$ const& a, $client_class_name$ const& b) {\n"
-    "    return a.connection_ == b.connection_;\n"
-    "  }\n"
-    "  friend bool operator!=($client_class_name$ const& a, $client_class_name$ const& b) {\n"
-    "    return !(a == b);\n"
-    "  }\n"
-    "  //@}\n");
-  // clang-format on
+  HeaderPrint(
+      R"""(
+$class_comment_block$
+class $client_class_name$ {
+ public:
+  explicit $client_class_name$()""");
+  if (IsExperimental()) HeaderPrint("ExperimentalTag, ");
+  HeaderPrint(
+      R"""(std::shared_ptr<$connection_class_name$> connection, Options opts = {});
+  ~$client_class_name$();
+
+  //@{
+  // @name Copy and move support
+  $client_class_name$($client_class_name$ const&) = default;
+  $client_class_name$& operator=($client_class_name$ const&) = default;
+  $client_class_name$($client_class_name$&&) = default;
+  $client_class_name$& operator=($client_class_name$&&) = default;
+  //@}
+
+  //@{
```

Comment:
unrelated: these should be `///@{` to actually start a group, ditto for closing the group.

Reply:
Ack. I'll send a follow up PR that touches all the files.